enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/types.ts

```diff
@@ -65,4 +70,5 @@ export type InteractiveQuestionContextType = Omit<
     onSave: (question: Question) => Promise<void>;
   } & {
     isCardIdError: boolean;
+    originalId: InteractiveQuestionId;
```

Comment:
did we miss this before? Is this for completeness, but not necessary for PR to pass checks?

Reply:
@WiNloSt we were abusing the LoadQuestionHook's type as it has been merged with the context type. The hook never returned originalId and we've always passed it manually to the context.